rcsullivanii/flick-finder
# mysql connection + login flow

Connect MySQL movie_recommendation_app to App

To test:
ng serve to test front end
node index.js to connect to sql database (cd backend first)

Re:
Basically everything is placeholder or installations. index.js contains all backend logic (see lines 17 - 48 for database table architecture). We can change however we see fit but this covers log in and sign up for new users. 

APIs included are...
authenticate user (login component), create new user (signup component), see all users (not called), fetch list user's movies (not called), add to your recommendation list (not called)
